tox.ini

```diff
@@ -14,8 +15,6 @@ deps =
 commands =
     unit: pytest {posargs}
     flake8: flake8 {posargs}
-    pylint: python setup.py lint
+    pylint: sh -c "git ls-files -co --exclude-standard '*.py' | grep -Ev '^(utils|roles/lib_(openshift|utils)/src)/' | xargs pylint"
```

Comment:
I don't say this is particularly great, open to find improvements.

- I don't like to have to wrap on `sh -c`
- I don't like using `grep` inline like that -- would prefer an easier-to-read list of excludes

One thing I'd like to be able to do is to pass files as positional arguments to limit the scope during development:

```
tox -e py35-ansible22-pylint -- roles/new_role/library/file.py
```